PipeMethodCalls/Endpoints/PipeClient.cs

```diff
@@ -75,6 +72,22 @@ public PipeClient(string serverName, string pipeName, PipeOptions options, Token
 			this.impersonationLevel = impersonationLevel;
 			this.inheritability = inheritability;
 		}
+		
+		/// <summary>
+		/// Get the raw named pipe. This will automatically create if it hasn't been instantiated yet and is accessed.
+		/// </summary>
+		public NamedPipeClientStream PipeInstance

```

Comment:
Maybe call this RawPipe to differentiate this from our wrapper instance

Reply:
Good point - included in cleanup.